style.css

```diff
@@ -116,12 +162,44 @@ article h1 {
   margin: 0;
 }
 
+.about-me li {
+  margin: 0 10px 0 0;
+}
+
 .social-links ul {
   list-style: none;
   display: flex;
   padding-right: 10px;
 }
 
+.frame2 li {
+  margin-left: 30px;
+  font-family: 'Poppins' Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.tags li {
+  color: #3b4eff;
+  font: 500 12px;
+  padding: 0 2px 0 -21px;
+  background: #ebebff;
+  border-radius: 8px;
+  text-decoration: none;
+  margin: 0 15px 0 0;
+  display: flex;
+  justify-content: flex-start;
+}
```

Comment:
- [ ] Instead of you tried to implement the changes required by the last code reviewer they didn't work in the browser, I also cloned in the case the changes could look locally, but I had the same result, please fix them to be similar to the Figma template: 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/212355609-56f74091-9db5-47e3-a55c-157bbfca522f.png)

![image](https://user-images.githubusercontent.com/58642949/212355784-b5fb8c4e-c2b6-4802-90fd-35bfc3a19f68.png)

**FIGMA DESIGN**

![image](https://user-images.githubusercontent.com/58642949/212356096-8b16dda3-0ff5-4187-92fc-299b41403588.png)